examples/00-basic/00-cycles-to-failure.py

```diff
@@ -0,0 +1,103 @@
+"""
+
+DPF-Core Fatigue Engineering Simple Example
+~~~~~~~~~~~~~~~~~~~~
+This example shows how to generate and use a result file to calculate the 
+cycles to failure result for a simple model.
+
+Material data is manually imported, Structural Steel from Ansys Mechanical:
+Youngs Modulus (youngsSteel)
+Poisson's Ratio (prxySteel)
+Cycles to Failure (snData)
+
+The first step is to generate a simple model with high stress and save the 
+results .rst locally to myDir (default C:\temp).
+For this we use a short pyMapdl script
+
+The second step uses DPF-Core to generate the cycles to failure result.
+The locally saved rst is imported and plotted.
+Then the von Mises stress is generated and plotted with DPF operators.
+The python package numpy is then used to interpolate the cycles to failure values.
+The nodal von Mises equivalent stress value is used in the interpolation.
+Note the cycles to failure data needs to be manipulated to use numpy interpolation.
+
+An empty field is created and filled with the resulting cycles to failure values.
+Cycles to failure result plotted.
+
+The cycles to failure result is the (interpolated) negative of the stress result.
+The higher the stress result, the lower the number of cycles to failure.
+
+Start MAPDL as a service to generate the rst file
+and import the DPF-Core module as ``dpf_core``.
+"""
+
+
+from ansys.mapdl.core import launch_mapdl
+from ansys.dpf import core as dpf
+import numpy as np
+myDir = r'c:\temp'
```

Comment:
It's best not to specify a directory. By default, PyMAPDL creates a temporary directory that's OS independent. It can be accessed simply by:

```py
mapdl.directory
```